File: Day-02/solution.py
input_str = input('Enter a string: ')

if not input_str.strip():
    print("Empty string!")
    exit()

str_len = len(input_str)
reverse_str = ''

for i in range(str_len - 1, -1, -1):
    reverse_str += input_str[i]

print(f"Reverse string: {reverse_str}")

File: Day-02/solution.js
import readline from 'readline';

const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});

rl.question('Enter a string: ', (inputStr) => {
    if (inputStr.trim().length === 0) {
        console.log("Empty string!");
        rl.close();
        return;
    }

    const strLen = inputStr.length;
    let reverseStr = '';

    for (let index = strLen - 1; index >= 0; index--) {
        reverseStr += inputStr[index];
    }

    console.log(`Reverse string: ${reverseStr}`);

    rl.close();
});
